v1/python-sdk/tutorials/automl-with-azureml/forecasting-backtest-single-model/assets/retrain_models.py

```diff
@@ -126,23 +136,18 @@ def run_backtest(data_input_name: str, file_name: str, experiment: Experiment):
         )
         print(f"The model {best_run.properties['model_name']} was registered.")
 
-    _, x_pred = fitted_model.forecast(X_test)
-    x_pred.reset_index(inplace=True, drop=False)
-    columns = [automl_settings[constants.TimeSeries.TIME_COLUMN_NAME]]
-    if automl_settings.get(constants.TimeSeries.GRAIN_COLUMN_NAMES):
-        # We know that fitted_model.grain_column_names is a list.
-        columns.extend(fitted_model.grain_column_names)
-    columns.append(constants.TimeSeriesInternal.DUMMY_TARGET_COLUMN)
-    # Remove featurized columns.
-    x_pred = x_pred[columns]
-    x_pred.rename(
-        {constants.TimeSeriesInternal.DUMMY_TARGET_COLUMN: "predicted_level"},
-        axis=1,
-        inplace=True,
-    )
+    # By default we will have forecast quantiles of 0.5, which is our target
+    if forecast_quantiles:
+        if 0.5 not in forecast_quantiles:
+            forecast_quantiles.append(0.5)
+        fitted_model.quantiles = forecast_quantiles
+
+    x_pred = fitted_model.forecast_quantiles(X_test)
+    x_pred.rename({0.5, "predicted_level"}, axis=1, inplace=True)
```

Comment:
This requires rework.

Can fail when forecast_quantiles are not supplied. Also, we need to make sure that 0.5 is always in forecast_quantiles if we want to only rely on ``forecast_quantiles`` method.

Reply:
forecast_quantiles by default should provide the 0.5. Hence, there is no need to specify it.

Reply:
resolving this as per confirmation from @vlbejan.

@SamGos93, please check the failing workflow.